projects/Mallard/src/components/article/article-header/review-header.tsx

```diff
@@ -49,6 +50,7 @@ const ReviewHeader = ({
                 {image ? (
                     <ArticleFader buildOrder={1}>
                         <ArticleImage
+                            proxy={<Stars rating={4} />}
```

Comment:
We have this in the API now don't we, can we rebase form master and use the proper stars before merging? Otherwise this looks 👍

Reply:
yup!!

Reply:
💣 - hit me up with the re-review afterwards (or msg me)

Reply:
there ya go